### PR TITLE
Feature/max polling frequency strategy

### PR DESCRIPTION
**Pull request by garygitton**
I added the change following the last conversation.
- align => and =
- removed strategy from default ones
- add docblock

### CODE REVIEW

**Comment by bakura10** [comment]
Travis is failing. Can you fix it?

File: src/SlmQueue/Strategy/MaxPollingFrequencyStrategy.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     * @return void
+     */
+    public function onQueueProcessFinish(WorkerEvent $event)
+    {
+        $startTime  = microtime(true);
+        $time       = ($startTime - $this->lastTime);
+
+        $minTime = 1 / $this->maxFrequency;
+
+        if($time < $minTime) {
```
Comment (bakura10) [inline src/SlmQueue/Strategy/MaxPollingFrequencyStrategy.php]: That's the part that makes Travis unhappy. You need to add a space after if.

**Comment by bakura10** [comment]
Once Travis is ok, this will look ok to me :).

**Comment by bakura10** [comment]
Thanks. @juriansluiman could you do a latest quick review before I merge?

**Comment by bakura10** [comment]
If no  news from jurian until next Wednesday I'll merge. @garygitton can you remind me if I forget ?

**Comment by garygitton** [comment]
No problem, I'll.

**Comment by garygitton** [comment]
Hi @bakura10, 
Just to remind you to merge the branch.
Have a good weekend,
Gary

**Comment by bakura10** [comment]
Oops... This has been merged as 0.4.1. Thanks!

**Comment by garygitton** [comment]
Thank you Michael

**Pull request merged**